clients/admin-ui/src/pages/GVLNotice.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Link, Text } from "@fidesui/react";
```

Comment:
hm why is this under `src/pages`? for Next, that will automatically generate a page for this! whereas I think this is just a regular ol' component